### PR TITLE
Sasslint

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes

Use [Sasslint](https://github.com/DFurnes/sasslint) to lint our stylesheets. Currently has two linting rules, but more to come!

For review: @DoSomething/front-end

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/.sasslint.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/.sasslint.json:1]: Didn't you add this file in the other PR?

File: lib/themes/dosomething/paraneue_dosomething/Gruntfile.js [hunk 2]
```diff
@@ -229,7 +236,7 @@ module.exports = function(grunt) {
 
   // > grunt test
   // Run included unit tests and linters.
-  grunt.registerTask('test', ['eslint']);
+  grunt.registerTask('test', ['eslint', 'sasslint']);
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/Gruntfile.js:239]: This to?

File: lib/themes/dosomething/paraneue_dosomething/.sasslint.json (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/.sasslint.json:1]: yup yup yup, those were in forge!

Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/.sasslint.json:1]: lol OOOO duh.

**Comment by sbsmith86** [comment]
:100:

**Pull request merged**